t3f/decompositions_test.py

```diff
@@ -71,8 +108,7 @@ def testTTMatrix(self):
     mat = np.random.rand(np.prod(out_shape), np.prod(inp_shape))
     mat = mat.astype(np.float32)
     tf_mat = tf.constant(mat)
-    tt_mat = decompositions.to_tt_matrix(tf_mat, (out_shape, inp_shape),
-                                         max_tt_rank=90)
+    tt_mat = decompositions.to_tt_matrix(tf_mat, (out_shape, inp_shape), max_tt_rank=90)
```

Comment:
Why did you make it single line? I think it is above the 80 symbols limit this way.